tests/lean/traceTacticSteps.lean.expected.out

```diff
@@ -5,30 +5,30 @@
     by
       skip
       trivial
+      
   [Elab.step.result] ?m
 [Elab.step] skip
 [Elab.step] 
       skip
       trivial
+      
 [Elab.step] 
       skip
       trivial
+      
```

Comment:
It seems the formatter regressed here?

Reply:
This is fun.  It turns out that `tacticSeq` now parses the trailing newline (after `trivial`) as a separator.  And the formatter then prints this final newline-separator.  We might want to change the formatter to drop the final separator if it's a newline.

Reply:
Final newline tweak implemented.